examples/viewer.html

```diff
@@ -380,6 +408,14 @@
     };
 
 
+    Viewer.prototype.sendRequest_ = function(type, data, awaitResponse) {
+      if (!this.messaging_) {
```

Comment:
will we need to do some queuing here at some point? i know messaging will be there after handshake, but im not sure if the current changes solve the "the viewer fired visibility change but the Viewer service isnt ready" issue. (still have that ticket open)